server/middleware/auth-check.js

```diff
@@ -27,6 +26,7 @@ module.exports = (req, res, next) => {
         return res.status(401).end();
       }
 
+      req.user = {};
       return next();
     });
   });
```

Comment:
I have a doubt here. The auth token being sent  in the packet header can be obtained on the wire and used to simulate another user. Is packet encryption [.. https] the only way around to do this??

Reply:
How do you imagine this happening ?

Reply:
I have searched on the internet and I don't think there is any other option other than using HTTPS. Actually not only the token, even the password is being sent in plain text to the server, so that can be obtained by a man in the middle attack too.

Reply:
@devanshu0987 Someone may inspect the packets being sent and will get the token since the token is being sent unencrypted.

Reply:
@devanshu0987  wireshark duh XD